Makefile

```diff
@@ -138,8 +138,15 @@ run/site:
 # Lint commands
 
 .PHONY: lint/go
+lint/go: FIX ?= false
+lint/go: VERSION ?= sha256:78d1bbd01a9886a395dc8374218a6c0b7b67694e725dd76f0c8ac1de411b85e8 #v1.46.2
+lint/go: FLAGS ?= --rm -e GOLANGCI_LINT_CACHE=/repo/.cache/golangci-lint -v ${PWD}:/repo -w /repo -it
 lint/go:
-	@echo "Unimplemented"
+ifeq ($(FIX),true)
+	docker run ${FLAGS} golangci/golangci-lint@${VERSION} golangci-lint run --fix
```

Comment:
Should we add `--rm` to remove the container after the job is done? 🤔

Reply:
this one?
https://github.com/pipe-cd/pipecd/pull/4086/files#diff-76ed074a9305c04054cdebb9e9aad2d818052b07091de1f20cad0bbac34ffb52R143

Reply:
Ah true, I missed reading that part 👍